src/_style.scss

```diff
@@ -456,3 +456,7 @@ select.form-control {
   line-height: 2rem;
   font-weight: 400;
 }
+
+.alert-message {
```

Comment:
No need for custom class, keep the font size as it is. If it needs to be changed, it should be changed in paragon